PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1232,6 +1233,10 @@ Do you want to launch Notepad++ in Administrator mode?"/>
 Notepad++ will be restarted after all the operations are terminated.
 Continue?"/>
             <NeedToRestartToLoadPlugins title="Notepad++ need to be relaunched" message="You have to restart Notepad++ to load plugins you installed."/> <!-- HowToReproduce: Import a plugin via menu "Settings->Import->Import Plugin(s)...". -->
+            <SaveAllConfirm title="Are you sure?" message="Are you sure you want to save all documents?
```

Comment:
The caption should not be "Are you sure?" but rather "Save All Confirmation" since the option name is "Enable Save All confirm dialog"